src/columns.h

```diff
@@ -92,7 +92,6 @@ inline cpp11::list create_columns(
     size_t num_threads) {
 
   R_xlen_t num_cols = idx->num_columns();
-  auto num_rows = idx->num_rows();
```

Comment:
This line and those removed below date back to 04cc4fc.

`num_rows` is only used in the lines removed below, so it becomes an unused variable and generates a compilation warning once we remove those lines.